compose.yaml

```diff
@@ -0,0 +1,9 @@
+
+services:
+  server:
+    build:
+      context: .
+    environment:
+      NODE_ENV: production
+    ports:
+      - 3000:3000
```

Comment:
possibly add `env_file: .env.production`  in server. So that it is accesible while building and running the container